### ISSUE TITLE
'accout details' entry in panel-applet user@cinnamon.org invisible

### ISSUE THREAD

**Issue by oberon2007** [linuxmint/Cinnamon#7471]
```
 * Cinnamon version 3.6.6
 * Distribution - Manjarolinux
 * Graphics hardware: multiple. reported by several users, using a variety of hardware and drivers
 * 32 or 64 bit - 64bit, possibly both
 ```

**Issue**
As specified in the header and shown in the screenshot, the 'account details' entry in the panel-applet user@cinnamon.org is invisible, while still functional when clicked.

**Steps to reproduce**
Open the applet. Look at it. See that the top entry is missing.

**Expected behaviour**
The entry should not only react to clicking but also be visible, meaning it should display an icon and text.

**Other information**
screenshot: 
![screenshot_2018-04-15_21-15-28](https://user-images.githubusercontent.com/11800714/38783191-3e150924-40ff-11e8-9910-b6211016f305.png)

**Comment by oberon2007** [comment]
```
$ cinnamon --replace
Cinnamon warning: Software rendering detected: llvmpipe (LLVM 6.0, 256 bits)
Cjs-Message: 20:53:01.252: JS LOG: About to start Cinnamon
Cjs-Message: 20:53:01.252: JS LOG: ACTIVATING SOFTWARE RENDERING
St-Message: 20:53:01.378: cogl npot texture sizes SUPPORTED
Cjs-Message: 20:53:01.385: JS LOG: Cinnamon started at Sun Apr 15 2018 20:53:01 GMT+0000 (UTC)
Cjs-Message: 20:53:01.841: JS LOG: network applet: Cannot find connection for active (or connection cannot be read)
openGL version 3.3 detected (GL3 Cogl Driver)
MetaSyncRing disabled: couldn't find required GL extensions, or the minimum safe openGL version was not met
Cjs-Message: 20:53:02.335: JS LOG: Invalid network device type, is 14
Cjs-Message: 20:53:02.335: JS LOG: network applet: Found connection for active
```

**Comment by oberon2007** [comment]
This is from a virtual machine. Therefor the software rendering mode.
But as stated before, the issue is reported by multiple users on different hardware.

**Comment by JosephMcc** [comment]
When running `cinnamon --replace` do you see any new errors when opening that menu?

**Comment by oberon2007** [comment]
Nothing additional, no. Only line I sometimes see additionally is this:
```
Cinnamon warning: Log level 16: STACK_OP_REMOVE: window 0x65 not in stack
```

**Comment by oberon2007** [comment]
I mean. This line also shows before I open the applet.

**Comment by itzexor** [comment]
The error indication in cs_applets.py is probably a red herring. If I manually check over dbus the same way if it's running after I add it, then it shows running. That is probably just a [very reproducible] timing issue.

**Comment by oberon2007** [comment]
Maybe some kind of a permission issue?
Our latest update included **systemd**, **polkit** and **accountsservice**

**Comment by itzexor** [comment]
It looks like there were quite a few changes in accountsservice, and for whatever reason we're no longer reliably getting `notify::is_loaded` signal that tells us it's ready for us to read the properties.

**Comment by leigh123linux** [comment]
Works ok here

```
rpm -q accountsservice
accountsservice-0.6.42-9.fc28.x86_64
```

**Comment by oberon2007** [comment]
Well, we're already at **accountsservice 0.6.46** now.

**Comment by itzexor** [comment]
This was caused by a typo in our code. The changes that exposed it are only in the latest accountsservice 0.6.46

copied from commit message:
Upstream fixed 'changed' signal being emitted far too often which broke
initial loading of the user info here and exposed this typo. It would
still load and work as usual after the first changed signal.

**Comment by itzexor** [comment]
@oberon2007 if you're the maintainer you can just add that patch to the package to fix it for now. the fix won't be in a release until 3.8

**Comment by oberon2007** [comment]
I've done that already! Works perfectly. Thank you so much! :tada:
